src/SDKs/Batch/DataPlane/Azure.Batch/Common/Enumerations/TaskCountValidationStatus.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.Batch.Common
+{
+    /// <summary>
+    /// Whether the task counts have been validated.
+    /// </summary>
+    public enum TaskCountValidationStatus
+    {
+        /// <summary>
+        /// The task counts have been validated and are guaranteed to match the List Tasks results.
```

Comment:
yea... actually apparently they return `Validated` even if they haven't run the validator yet. They only return `Unvalidated` if the validator fails... so calling it `Validated` is a bit of a stretch... (Though that is what it's called in the API).